src/legacy/core_plugins/kibana/public/management/sections/settings/components/field/field.tsx

```diff
@@ -764,6 +774,7 @@ export class Field extends PureComponent {
               helpText={this.renderHelpText(setting)}
               describedByIds={[`${setting.name}-aria`]}
               className="mgtAdvancedSettings__fieldRow"
+              // @ts-ignore
               hasChildLabel={setting.type !== 'boolean'}
```

Comment:
Can we cast here a variable to the required type, instead of using `// @ts-ignore`.

Reply:
This is here due to incomplete eui typescript type defs.